Formula/libvirt.rb

```diff
@@ -14,6 +14,7 @@ class Libvirt < Formula
 
   depends_on "pkg-config" => :build
   depends_on "gnutls"
+  depends_on "curl"
```

Comment:
... and now the failure isn't occurring.

I have no idea what was causing it the other day, but curl doesn't seem to be needed after all. 😵 

Will push a new commit in a sec.